src/backend/matrix_client.py

```diff
@@ -1136,6 +1136,17 @@ async def room_join(self, alias_or_id_or_url: str) -> str:
         response = await super().join(string)
         return response.room_id
 
+    async def toggle_bookmark(self, room_id: str) -> None:
+        room = self.models[self.user_id, "rooms"][room_id]
+        room.bookmarked = not room.bookmarked
+
+        bookmarks = self.backend.ui_settings["roomBookmarkIDs"].setdefault(self.user_id, [])
```

Comment:
```suggestion
        settings  = self.backend.ui_settings
        bookmarks = settings["roomBookmarkIDs"].setdefault(self.user_id, [])
        
```
To keep it under 80 columns